openoffload/cpp/framework/build/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM centos:8.4.2105
+FROM rockylinux:9
```

Comment:
I would use fedora... like we do in other repos...

Reply:
Fedora isn't CentOS, Rocky Linux is what CentOS used to be, so I will ignore this comment. See https://rockylinux.org

Reply:
Or `quay.io/centos/centos:stream9`